static/src/javascripts/projects/common/utils/referrers.js

```diff
@@ -0,0 +1,7 @@
+define(function () {
+    return {
+        isFacebook: function () {
+            return /\.facebook\.com/.test(document.referrer);
```

Comment:
Would be good to add `isFacebookReferral` and `isTwitterReferral` methods to the [detect](https://github.com/guardian/frontend/tree/master/static/src/javascripts/projects/common/utils) util, then [refactor this other usage](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/ui/sticky-social.js#L91)